samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/ModelScorer/TFModelScorer.cs

```diff
@@ -66,14 +66,8 @@ public void Score()
             Console.WriteLine($"Training file: {dataLocation}");
             Console.WriteLine($"Default parameters: image size=({ImageNetSettings.imageWidth},{ImageNetSettings.imageHeight}), image mean: {ImageNetSettings.mean}");
 
-            TextLoader loader = mlContext.Data.CreateTextReader(
-                                                    columns: new[] 
-                                                                {
-                                                                  new TextLoader.Column("ImagePath", DataKind.Text, 0),
-                                                                });
-
-            var data = loader.Read(new MultiFileSource(dataLocation));
-
+            var data = mlContext.Data.ReadFromTextFile<ImageNetData>(dataLocation, hasHeader: true);
```

Comment:
@CESARDELATORRE  I see we don't need to change this sample. Let's see comment in readme file below

"It is important to highlight that the label in this is not really used when scoring with the TensorFlow model. It is in this file only as a reference when testing the predictions so you can compare the actual label of each sample data with the predicted label provided by the TensorFlow model. That is why when loading the file with the 'TextLoader' above you just taking the ImagePath or name of the file but you are not taking the label."

As it is mentioned that label is not mentioned while using TextLoader, I want to keep this sample with the previous changes.i.e using the TextLoader to read the schema.

Please suggest your comments if I can change the sample with the previous changes or not.